os/platformcommondefines.h

```diff
@@ -34,50 +34,8 @@ VkFormat NativeToVkFormat(int native_format);
 // minigbm specific DRM_FORMAT_YVU420_ANDROID enum
 #define DRM_FORMAT_YVU420_ANDROID fourcc_code('9', '9', '9', '7')
 
-/*
```

Comment:
We have removed this patch in libdrm, master should be updated to use the latest libdrm